ui/core-react/src/core-react/utils/Point.ts

```diff
@@ -6,38 +6,38 @@
  * @module Utilities
  */
 
-import type { PointProps } from "@itwin/appui-abstract";
+import type { XAndY } from "@itwin/core-geometry";
```

Comment:
There is something off in the `package.json` because `core-react` do not depend on `core-geometry`, yet it is there in `devDependencies`. On top of that, these are internal classes, and we already have defined `PointProps` in `popup/Popup.tsx`, can you move the definition in a `uils/PointProps.tsx`, but do not put that file in the barrel export, so it remains internal.